service-base/src/main/java/org/eclipse/hono/service/monitoring/LoggingConnectionEventProducer.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+package org.eclipse.hono.service.monitoring;
+
+import static java.lang.invoke.MethodType.methodType;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+
+import org.eclipse.hono.service.auth.device.Device;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+/**
+ * A logging only implementation.
+ */
+public class LoggingConnectionEventProducer implements ConnectionEventProducer {
+
+    private static final Logger logger = LoggerFactory.getLogger(LoggingConnectionEventProducer.class);
+
+    private final MethodHandle handle;
+
+    /**
+     * Create a new instance with the default log level ({@link Level#INFO}).
+     */
+    public LoggingConnectionEventProducer() {
+        this(null);
+    }
+
+    /**
+     * Create a new instance with the provided log level.
+     * 
+     * @param level The provided log level to use, may be {@code null} in which case {@link Level#INFO} will be used.
+     */
+    public LoggingConnectionEventProducer(Level level) {
```

Comment:
I thought we configured log levels by means of log config files.

Reply:
True. We could drop this and let people configure this using the log system configuration. I would suggest to switch to "debug" in this case in order to not spam the log if people don't want this.